Source/WebCore/loader/TextResourceDecoder.cpp

```diff
@@ -36,35 +36,25 @@
 #include <wtf/text/MakeString.h>
 #include <wtf/text/ParsingUtilities.h>
 
-WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
-
 namespace WebCore {
 
 using namespace HTMLNames;
 
-static constexpr bool bytesEqual(const uint8_t* p, uint8_t b)
-{
-    return *p == b;
-}
-
-template<typename... T>
-static constexpr bool bytesEqual(const uint8_t* p, uint8_t b, T... bs)
-{
-    return *p == b && bytesEqual(p + 1, bs...);
-}
+constexpr std::array<uint8_t, 3> bom = { 0xEF, 0xBB, 0xBF };
```

Comment:
```suggestion
constexpr std::array<uint8_t, 3> byteOrderMarkUTF8 = { 0xEF, 0xBB, 0xBF };
```
This is a BOM encoded with UTF-8. Also the type of the array is not even `uchar8_t`, the type for UTF-8 characters. So I think the name should include UTF-8 in it.